sklearnex/svm/_classes.py

```diff
@@ -0,0 +1,255 @@
+#===============================================================================
```

Comment:
I think need separate on svc.py and svr.py files. Or rename to svm.py

Reply:
Don't we want to be consistent with scikit-learn in files' names?

Reply:
Well, they have a classes_ with the context of svm.  why be 1 in 1 in such places?